### ISSUE TITLE
Question:  How augment like in RandomResizedCrop with imgaug?

### ISSUE THREAD

**Issue by EliasVansteenkiste** [aleju/imgaug#662]
How would you define the RandomResizedCrop torchvision.transforms with imgaug?
RandomResizedCrop is commonly used for training on Imagenet.
There are several crop options in imgaug, but I don't see an exact match.

Reference:
https://pytorch.org/docs/stable/torchvision/transforms.html#torchvision.transforms.RandomResizedCrop

**Comment by EliasVansteenkiste** [comment]
I tried to mimic the functionality with the following sequence but it's hard to control the aspect ratio.

```
iaa.Sequential([
            iaa.size.Crop(percent=((0, .3), (0, .3), (0, .3), (0, .3))),
            iaa.size.PadToSquare(pad_mode=imgaug.ALL),
            iaa.Resize({"height": 224, "width": 224}),
            iaa.Fliplr(p=.5)
        ])
```

**Comment by aleju** [comment]
I don't think that there is an exact replacement for that operation in imgaug. You would have to use a custom crop operation, followed by `Resize`. The easiest way to implement the custom operation is probably to subclass `CropToFixedSize` and overwrite its `_draw_samples()` method. Would still be a bit of work though.
